### DISCUSSION TITLE
Job queue

### DISCUSSION THREAD

**Discussion by wutipong** [bmaltais/kohya_ss#1620] [Ideas]
Currently the UI perform the task immediately when press 'start'. If there's existing task running at that time the console would just display an error. 

I think it would be nice if I can queue up some LoRA training tasks (which might take a little over 1 hour) to run over night rather than have to start each tasks manually.

**Comment by Syscore64** [comment]
Yes pleace and give the Option: Loading multiple jsons. 
Job1 done -> load job2 - end this -> load job3

this is what i really need - pleace

**Comment by manofthemachine** [comment]
has this been implemented yet?

**Reply by Syscore64** [reply]
Dont see any in the Changelog or in the UI :/
